docs/user_guide/import_workflow.md

```diff
@@ -138,7 +138,19 @@ Then, with a simple click you generate your project. All the toppings received f
 
 ![baked project](../assets/generated_project.png)
 
-Bon appetit!
+### TID (OID) Values
+
+Often the models definition requires cross-system unique identificators. So called OIDs or TIDs, what are represented in the physical database as the `t_ili_tid` column. Find a clear definition and more details about them in the [corresponding chapter](../../background_info/oid_tid_generator).
+
+On creating a QGIS Project with Model Baker, there are default default value expression generated for the `t_ili_tid` field on the attribute form. But often those default value expressions need to be edited by the user (like e.g. the prefix in the `STANDARDOID`).
```

Comment:
I guess the double default in "default default value expression" is on purpose, would it make sense to replace the first one with "preset" or reformulate to "a default value expression is added by Model Baker"?

Reply:
yes on pupose and yes confusing. changed it...